cmd/initialize/initlizeMonorepo.go

```diff
@@ -41,27 +41,55 @@ func runInitMonorepo(cmd *cobra.Command, args []string) {
 	initproject.IsMonorepo = true
 	initproject.RunInitProject(cmd, args)
 
-	// Get apps to initialize
-	appsOfMonorepoDir, err := prompt.PromptForMonorepoApps(cmd)
-	if err != nil {
-		printer.Error(cmd, err)
-		return
-	}
-
 	m, err := manifest.Restore()
 	if err != nil {
 		printer.Error(cmd, err)
 		return
 	}
 
-	// Initialize each app
-	for _, appDir := range appsOfMonorepoDir {
-		printer.Info(fmt.Sprintf("Initializing app %s", appDir))
-		err := initializeMonorepoApp(cmd, appDir, orgID, m.Config, appService, envService, m.Secret)
+	for {
+		appPath, err := prompt.PromptString(cmd, "Provide the path to an application:")
+		if err != nil {
+			printer.Error(cmd, err)
+			return
+		}
+
+		cleanPath := filepath.Clean(appPath)
+
+		if strings.HasPrefix(cleanPath, "..") || strings.Contains(cleanPath, "../") {
+			printer.Error(cmd, fmt.Errorf("invalid path: cannot reference parent directories"))
+			continue
+		}
+
+		currentDir, err := os.Getwd()
 		if err != nil {
-			printer.Error(cmd, fmt.Errorf("failed to initialize app %s: %w", appDir, err))
+			printer.Error(cmd, fmt.Errorf("failed to get current directory: %w", err))
+			return
+		}
+
+		absPath, err := filepath.Abs(cleanPath)
+		if err != nil {
+			printer.Error(cmd, fmt.Errorf("invalid path: %w", err))
+			continue
+		}
+
+		// Ensure the path is under current directory
+		if !strings.HasPrefix(absPath, currentDir) {
+			printer.Error(cmd, fmt.Errorf("invalid path: must be within current directory"))
 			continue
 		}
+
+		printer.Info(fmt.Sprintf("Initializing app %s", cleanPath))
+		err = initializeMonorepoApp(cmd, cleanPath, orgID, m.Config, appService, envService, m.Secret)
+		if err != nil {
+			printer.Error(cmd, fmt.Errorf("failed to initialize app %s: %w", cleanPath, err))
+			continue
+		}
+
+		moreApps := prompt.PromptYesNo(cmd, "Do you have another?", false)
```

Comment:
@nathanyoung can you review these commands for verbiage/voice/tone ?

Reply:
If i'm understanding what's happening correctly... let's be consistent and use sentence case (capitalize first letter in first word). 

So instead of `failed to get current...` use `Failed to get current...`, etc

Reply:
@nathanyoung I think the question is specifically about this one line - if the user has multiple apps they are initializing within a monorepo we ask after each one. The prompt here is `Do you have another?` which the user must respond with `y/n` to.